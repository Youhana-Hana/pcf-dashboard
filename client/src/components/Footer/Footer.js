import React, { Component } from 'react';

class Footer extends Component {
  render() {
    return (
      <footer className="app-footer">
        <a href="#"></a>&copy; 2017 Bosch IoT Cloud.
                                           <span className="float-right">Powered by <a href="#">Bosch</a></span>
      </footer>
    )
  }
}

export default Footer;
