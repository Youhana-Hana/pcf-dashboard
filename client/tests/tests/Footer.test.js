import React from 'react';
import renderer from 'react-test-renderer';
import Footer  from '../../src/components/Footer/';

it('test to see if  renders correctly', () => {
  const footer = renderer
  .create(<Footer />)
  .toJSON();
  expect(footer).toMatchSnapshot();
});
