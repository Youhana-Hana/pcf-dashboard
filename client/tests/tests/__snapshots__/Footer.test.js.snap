// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test to see if  renders correctly 1`] = `
<footer
  className="app-footer"
>
  <a
    href="#"
  />
  Â© 2017 Bosch IoT Cloud.
  <span
    className="float-right"
  >
    Powered by 
    <a
      href="#"
    >
      Bosch
    </a>
  </span>
</footer>
`;
